cmd/autoowners/main.go

```diff
@@ -22,10 +22,6 @@ import (
 )
 
 const (
-	doNotEdit     = "# DO NOT EDIT; this file is auto-generated using tools/populate-owners."
```

Comment:
I mean we can keep this -- (and update the tool name), just not make it commit-specific, so it does not change.

Reply:
Done.